src/job_metadata.c

```diff
@@ -555,6 +557,8 @@ TupleToCronJob(TupleDesc tupleDescriptor, HeapTuple heapTuple)
 								  tupleDescriptor, &isNull);
 	Datum userName = heap_getattr(heapTuple, Anum_cron_job_username,
 								  tupleDescriptor, &isNull);
+	Datum active = heap_getattr(heapTuple, Anum_cron_job_active,
+                                                                  tupleDescriptor, &isNull);
```

Comment:
It would be good to make this robust to older catalogs for when the binaries get automatically upgraded, but the ALTER EXTENSION did not run yet. Otherwise, this code would crash the server, which gets pretty nasty if you created a `@reboot` job.

You could do:
```
if (HeapTupleHeaderGetNatts(heapTuple->t_data) >= Anum_cron_job_active)
{
    active = heap_getattr(...);
}
```

Reply:
Thanks for the hint. I've added it as well https://github.com/citusdata/pg_cron/pull/47/commits/0305160449e4ea2b7f4862ad88b26f055d7fbc74.